networks/suzuka/suzuka-client/src/tests/mod.rs

```diff
@@ -135,4 +141,18 @@ async fn test_example_interaction() -> Result<()> {
     );
 
     Ok(())
+}
+
+#[cfg(feature = "load-soak-hey-partners")]
+fn load_soak_hey_partners() {
+	println!("Initialize movementswap test...");
+
+	let config = ExecutionConfig::default();
+	config.logfile_path = "movementswap.log".to_string();
+	config.execfile_path = "~/.movement/movementswap-core/test.sh"to_string();
+	if let Err(err) = init_test(&config) {
```

Comment:
Probably want to look at the error handling here.  We fail to initialise and then continue to execute.  `init_test(&config)?` could be clearer